src/current/v24.1/molt-fetch.md

```diff
@@ -275,23 +309,116 @@ To drop existing tables and create new tables before loading the data, use `'dro
 --table-handling 'drop-on-target-and-recreate'
 ~~~
 
-With each option, MOLT Fetch creates a new CockroachDB table to load the source data if one does not exist.
+When using the `'drop-on-target-and-recreate'` option, MOLT Fetch creates a new CockroachDB table to load the source data if one does not already exist. To guide the automatic schema creation, you can [explicitly map source types to CockroachDB types](#type-mapping).
+
+#### Type mapping
+
+If [`'drop-on-target-and-recreate'`](#target-table-handling) is set, MOLT Fetch automatically creates a CockroachDB schema that is compatible with the source data. The column types are determined as follows:
+
+- PostgreSQL types are mapped to existing CockroachDB [types]({% link {{ page.version.version }}/data-types.md %}) that have the same [`OID`]({% link {{ page.version.version }}/oid.md %}).
+- The following MySQL types are mapped to corresponding CockroachDB types:
+
+	|                      MySQL type                     |                                                CockroachDB type                                                |
```

Comment:
I think if you including leading and trailing `|` characters, you will see a weird artifact on the left-hand side right after the table. This doesn't happen if you remove the leading and trailing pipes, and as a bonus you don't have to maintain the width / alignment of the last column manually. In the style guide, we mention not to use leading and trailing pipes.

Reply:
Noted... I'm using a plugin that auto-formats the tables emacs-style, which is why these are here & I don't have to worry about manual alignment.